docs/src/rules/no-unused-vars.md

```diff
@@ -105,23 +107,45 @@ Note that `/* exported */` has no effect for any of the following:
 
 The line comment `// exported variableName` will not work as `exported` is not line-specific.
 
-Examples of **correct** code for `/* exported variableName */` operation:
+```js
+/* exported global_var */
+
+var global_var = 42;
+```
+
+Examples of **correct** code for `/* exported variableName */` operation with `no-unused-var`:
 
 ::: correct
 
 ```js
+/*eslint no-unused-vars: "error"*/
 /* exported global_var */
 
+// rule will not report global_var
+// sourcType must be set to script
+// environment should not be node or commonjs
+// globalReturn must be set to false
+
```

Comment:
```suggestion
```

I think there's no need for this as the environment (and thus implicitly the ESLint configuration) where this applies is already explained at the beginning of this section.

Reply:
Done!